app/javascript/components/Download/PageListLayout.tsx

```diff
@@ -0,0 +1,56 @@
+import { Slot } from "@radix-ui/react-slot";
+import * as React from "react";
+
+import { classNames } from "$app/utils/classNames";
+
+export const PageListLayout = ({
+  pageList,
+  children,
+  className,
+}: {
+  pageList: React.ReactNode;
+  children: React.ReactNode;
+  className?: string;
+}) => (
+  <div
+    className={classNames(
+      "flex min-h-0 flex-col gap-x-16 gap-y-6 bg-background p-4 [scrollbar-gutter:stable] md:p-8 lg:flex-row lg:overflow-y-auto",
```

Comment:
```suggestion
      "flex min-h-0 flex-col lg:gap-16 gap-6 bg-background p-4 [scrollbar-gutter:stable] md:p-8 lg:flex-row lg:overflow-y-auto",
```
Think that makes the intent slightly more obvious, wdyt?